05/ContactForm.js

```diff
@@ -16,6 +18,7 @@ const ContactForm = () => {
     const formsReducer = (state, action) => {
         switch (action.type) {
             case UPDATE_FORM:
+                // eslint-disable-next-line no-case-declarations
                 const { name, value, hasError, error, touched, isFormValid } = action.data;
```

Comment:
Wystarczy przenieść to przed `switch`.
Z jakiegoś powodu te błędy są pokazywane przez `eslinta` i nie warto za każdym razem go wyłączać.